version.txt

```diff
@@ -0,0 +1 @@
+2.6.1
```

Comment:
I'd rather not have this file — it's yet another place we'll have to update when we bump the version. Why can't the make variables be used?

Reply:
OK. I will remove this file.